app/scripts/views/confirm.js

```diff
@@ -41,14 +44,23 @@ function (Cocktail, FormView, BaseView, Template, p, AuthErrors, Constants,
     },
 
     context: function () {
-      return {
+      var context = {
         email: this.getAccount().get('email')
       };
+
+      context.verificationExperiment = this._verificationExperiment.getContext();
+
+      if (this._verificationExperiment.is('openGmail')) {
+        this._verificationExperiment.performAction('triggered');
+      }
```

Comment:
We should probably be ultra explicit when logging which group a user is in. If the user is in the treatment group, that should be logged (obviously), but I think if the user is in the control group, that should be explicitly logged too. Doing so will make queries simpler.